bsp/stm32f407_renode/uart_retarget.c

```
@@ -0,0 +1,77 @@
+
+/* Minimal stubs for newlib syscalls that nosys.specs doesn't cover */
+int _close(int fd)                         { (void)fd; return -1; }
+int _fstat(int fd, struct stat *st)        { (void)fd; st->st_mode = S_IFCHR; return 0; }
+int _isatty(int fd)                        { (void)fd; return 1; }
+int _lseek(int fd, int offset, int whence) { (void)fd; (void)offset; (void)whence; return 0; }
+int _read(int fd, char *buf, int len)      { (void)fd; (void)buf; (void)len; return 0; }
+
+extern char end;
+static char *heap_end = 0;
+
+void *_sbrk(int incr) {
```

Cppcheck (2.20.0)

[style] The function '_sbrk' is never used.

(unusedFunction)